src/sandstorm/supervisor.h

```diff
@@ -97,6 +98,7 @@ class SupervisorMain: public AbstractMain {
   bool devmode = false;
   bool seccompDumpPfc = false;
   bool isIpTablesAvailable = false;
+  kj::Maybe<uid_t> setuidMode;
```

Comment:
I guess we call it the "setuid sandbox" here as an implementation detail, not just in strings that admins will see.

Still, I prefer if we call it something else to minimize admin confusion that we're using setuid root binaries, e.g the user-switching or uid-switch sandbox.